extensions/applicationinsights-analytics-js/rollup.config.js

```diff
@@ -20,126 +15,20 @@ const replaceValues = {
   "// Licensed under the MIT License.": ""
 };
 
-function doCleanup() {
-  return cleanup({
-    comments: [
-      'some', 
-      /^.\s*@DynamicProtoStub/i,
-      /^\*\*\s*@class\s*$/
-    ]
-  })
-}
+updateDistEsmFiles(replaceValues, banner, true, true, "dist-esm");
 
-const browserRollupConfigFactory = isProduction => {
-  const browserRollupConfig = {
-    input: `dist-esm/${outputName}.js`,
-    output: {
-      file: `browser/${outputName}.js`,
-      banner: banner,
-      format: "umd",
-      name: "Microsoft.ApplicationInsights",
-      extend: true,
-      freeze: false,
-      sourcemap: true
+export default simpleConfig(banner, 
```

Comment:
I don't remember why we have this "special" case (topLevel: true), but lets remove it so that all of the main plugins are the same.